modules/cron.js

```diff
@@ -2,8 +2,8 @@ const updateMembers = require( `${__dirname}/import-mailchimp.js` )
 const shell = require( 'shelljs' )
 
 // milliseconds * seconds * minutes hours
-let frequency = 1000 * 60 * 60 * 1 // Every hour
-console.log( process.argv[2] == 'once' ? 'Updating members once' : `Started member update cron every ${frequency/1000} seconds (${ frequency/1000/60/60 } hours)` )
+let frequency = ( process.env.frequency 1000 * 60 * 60 * 24 ) || 1000 * 60 * 60 * 1 // Every hour
```

Comment:
Will process.env.frequency evaluate to false if it isn't set?

Reply:
It evaluates to undefined. Due to the ```|| 1000 * 60 * 60 * 1 ``` statement the result of not specifying it ```frequency``` will equal 1 hour in milliseconds.
 
![screen shot 2018-03-09 at 13 01 24](https://user-images.githubusercontent.com/9071382/37206687-06b4c7b2-239a-11e8-8cdb-1901821dfc28.png)